.github/workflows/labeler.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
+on:
+  - pull_request_target
```

Comment:
shouldn't this be format like the following?

```suggestion
  pull_request_target:
```

Reply:
I copied this from the docs of the PR labeler action: https://github.com/actions/labeler/tree/v5.0.0-beta#create-workflow

Reply:
I always forget that the `on` block is allowed to be a list or map as well.

Reply:
![image](https://github.com/oauth2-proxy/oauth2-proxy/assets/18513179/cba62892-e067-4c00-bef8-5608d7bee620)

Reply:
> I always forget that the `on` block is allowed to be a list and map as well.

To keep it "simple" I guess... I do like the map option better though, after having another look